generic/tests/vlan.py

```diff
@@ -218,6 +246,42 @@ def flood_ping(src, dst):
             nc_transfer(test, 0, 1)
             nc_transfer(test, 1, 0)
 
+            if vlan == int(bridge_vlan):
+                error_context.context("Run netperf stress test between guests", logging.info)
+                utils_test.run_virt_sub_test(test, params, env, sub_type)
+                if vm.is_dead():
+                    vm.create(params=params)
+
+        error_context.context("Ping test among guests and host", logging.info)
+        guest_vlan = bridge_vlan
+        for vm_index, vm in enumerate(vms):
+            interface = ifname[vm_index] + '.' + str(guest_vlan)
```

Comment:
`guest_vlan = bridge_vlan`
`interface = ifname[vm_index] + '.' + str(guest_vlan)`
`bridge_vlan` is a str,  so you don't need to convert it again. 
There is no need to assign `bridge_vlan` to `guest_vlan`. You can use `bridge_vlan` directly, if you want to make it clear that `birdge_vlan` is `guest_vlan`, a comment will be better in this case.